teacher.rb

```diff
@@ -0,0 +1,11 @@
+# Teacher Class Inheriting From Person Class
+class Teacher < Person
+  def initialize(specialization)
+    super
+    @specialization = specialization
+  end
```

Comment:
- [ ] This method also should have the arguments from the parent, and "specialization" added to it.  Super should initialize the parent method with its required arguments like this:

```ruby
  def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
```
That way, the class will be properly initialized! 💪

Reply:
@Meltrust, Thank you so much for your review :star: